### PR TITLE
PDEs tutorial: Finite Differences, PINNs an Generalized Neuro-solver

### PR DESCRIPTION
**Pull request by hpenedones**
Reviving material from old blog post.

Staging website version:
https://tutorials.staging.inductiva.ai/

### CODE REVIEW

File: tutorials/pdes/heat-1-an-introduction.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# The Heat Equation
+
+**Authors**: Manuel Madeira, David Carvalho
+
+**Reviewers**: Fábio Cruz
+
+
+In this tutorial, we delve into the problem of ... 🔥 **heat diffusion** on a plate! 🔥
```
Comment (sarmento) [inline tutorials/pdes/heat-1-an-introduction.md:8]: Maybe you want to move a bit this text (or a sunmarized version of it) to the top page, as intro to the tutorial, and then have heat-1.md maybe repeat it again, with more explicit information about the structure (as it is). Maybe you also want to choose an image to put on the cover page of the tutorials.

File: tutorials/pdes/heat-1-an-introduction.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+This field is fast-growing but, most importantly is at its *blissful infancy* stage.
+Our dream, rooted in a belief, is that Machine Learning may be refined to excel
+in extremely complex and customizable cases.
+
+This sophistication could potentially fare better in performance than
+every other classical solver, allowing us to go to lands classical frameworks have never
+granted us so far!
+
+Alright. 🔥 *Let's get hot in here!* 🔥
```
Comment (sarmento) [inline tutorials/pdes/heat-1-an-introduction.md:32]: Yes: I think you can summarize this text a very shrort intro on the cover page.

Comment (sarmento) [reply tutorials/pdes/heat-1-an-introduction.md:32]: I also think we need to have a line like with something like this: 

"The world of classical simulation and the world AI are merging into a broader form of computational solutions for science and engineering. In a previous tutorial, we showed how you could use Inductiva's simulation API to generate synthetic data for training a GNN.In this tutorial, we will go in a slightly different direction and we guide you through an example one of the most exciting technical outcomes of this marriage between classical simulation and AI: Physically Informed Neural Networks, or PINNs for short. The most notable difference between using PINNs vs using more generic neural networks is that, in PINNs, we explicitly encode knowledge about the the physical laws in the training loss of the neural network, allowing it to more quickly converge to functions that are physically sound. 

For this tutorial, we will use a simple .... bla bla heat
The structure of this tutorial is..."

**Review by sarmento — APPROVED** [review]
I think this is mostly good. I left some suggestions.
Also, it seemed to me that some images are not centered on the page.


[NOT important] Also, my OCD would love all line to have at most 80 chars.

File: tutorials/pdes/heat-1-an-introduction.md (hunk 1 above)
Comment (hpenedones) [reply tutorials/pdes/heat-1-an-introduction.md:8]: thanks, done.

File: tutorials/pdes/heat-1-an-introduction.md (hunk 2 above)
Comment (hpenedones) [reply tutorials/pdes/heat-1-an-introduction.md:36]: Thanks!

**Comment by hpenedones** [comment]
> I think this is mostly good. I left some suggestions. Also, it seemed to me that some images are not centered on the page.
> 
> [NOT important] Also, my OCD would love all line to have at most 80 chars.

Ok, reformatted with "prettier" a tool that can enforce the 80 chars on markdown files... it messed a little bit with some sections containing math formulas of multiple lines, but it was easy to correct.
